bin/sbank_pipe

```diff
@@ -27,6 +27,7 @@ import socket
 import subprocess
```

Comment:
Is subprocess still needed at this point?

Reply:
It seems not, I've removed it as well/